scripts/west_commands/run_common.py

```diff
@@ -568,6 +567,12 @@ def load_cmake_cache(build_dir, args):
         log.die(f'no CMake cache found (expected one at {cache_file})')
 
 def rebuild(command, build_dir, args):
+    skip_rebuild_config = config.get('flash', 'skip-rebuild', fallback='false')
```

Comment:
Also suggest to use `getboolean`

```suggestion
    skip_rebuild_config = config.getboolean(command.name, 'skip-rebuild', fallback='false')
```